pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -124,34 +164,57 @@ class TelegramHandler(EventHandler):
     def __init__(self, bot, config):
         self.bot = bot
         self.tbot = None
-        self.master = config.get('master', None)
+        master = config.get('master', None)
         self.pokemons = config.get('alert_catch', {})
         self.whoami = "TelegramHandler"
         self.config = config
+        if master == None:
+            self.master = None
+            return
+
+        with self.bot.database as conn:
+            # if master is not numeric, try to fetch it from the database
+            if not re.match(r'^[0-9]+$', str(master)): # master is numeric
```

Comment:
> if str(master).isnumeric():
This does not work. isnumeric works only on unicode objects. If this line is really worth a change, then to "if unicode(master).isnumeric():". 

The difference is minimal though, not worth changing IMHO, unless a change is needed in that place anyway.